sample-install/snirouting_tls/base/kroxylicious-config.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kroxylicious-config
+data:
+  config.yaml: |
+    adminHttp:
+      endpoints:
+        prometheus: {}
+    virtualClusters:
+      demo:
+        targetCluster:
+          bootstrap_servers: my-cluster-kafka-bootstrap:9093
+          tls:
+            trust:
+              storeFile: /opt/kroxylicious/trust/ca.p12
+              storePassword:
+                filePath: /opt/kroxylicious/trust/ca.password
+        clusterNetworkAddressConfigProvider:
+          type: SniRouting
+          config:
+            bootstrapAddress: mycluster.kafka:9092
+            brokerAddressPattern: broker$(nodeId).mycluster.kafka
+        logNetwork: false
+        logFrames: false
+        tls:
+          key:
+            certificateFile: /opt/kroxylicious/server-tls/tls.crt
+            privateKeyFile: /opt/kroxylicious/server-tls/tls.key
```

Comment:
Best practice is that private keys have a password.  I'll re-configure cert-manager to write a password protected key file, or use a key store.